data/json/items/armor/suits_protection.json

```diff
@@ -1196,23 +1196,57 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str": "scrap suit" },
-    "description": "A suit of armor made from scraps of metal secured by simple strings; the loose collection of plates provides decent but not the most convenient protection.",
+    "description": "A suit of armor made from scraps of metal secured by simple strings; the loose collection of plates provides decent but uncomfortable protection.",
     "weight": "9534 g",
     "volume": "15500 ml",
     "price": 60000,
     "price_postapoc": 250,
     "to_hit": -5,
     "bashing": 6,
     "cutting": 8,
-    "material": [ "steel", "iron" ],
     "symbol": "[",
     "looks_like": "armor_lightplate",
     "color": "light_gray",
-    "warmth": 10,
     "longest_side": "40 cm",
-    "material_thickness": 2,
     "flags": [ "OUTER" ],
-    "armor": [ { "encumbrance": 18, "coverage": 80, "covers": [ "torso", "leg_l", "leg_r", "arm_l", "arm_r" ] } ]
+    "armor": [
+      {
+        "material": [
+          { "type": "budget_steel", "covered_by_mat": 75, "thickness": 2 },
+          { "type": "cotton", "covered_by_mat": 95, "thickness": 1.0 }
+        ],
+        "covers": [ "torso" ],
+        "coverage": 100,
+        "cover_melee": 80,
+        "cover_ranged": 80,
+        "cover_vitals": 0,
+        "encumbrance": 18
+      },
+      {
+        "material": [
+          { "type": "budget_steel", "covered_by_mat": 75, "thickness": 1.5 },
+          { "type": "cotton", "covered_by_mat": 95, "thickness": 1.0 }
+        ],
+        "covers": [ "arm_l", "arm_r" ],
```

Comment:
The non-suit version of this only covers the forearms so this is a weird upgrade, if all the scrap was sewn to a long sleeve shirt it would make sense but the, torso + arms + legs recipe earlier means this doesn't makes sense.

Reply:
This now includes rags to make a crude wrap, providing decent coverage and some slight warmth. This gives decent coverage and a bit of warmth while resolving the 'scrap metal against bare skin' problems.